packages/shared/src/utils/constants/contract.tsx

```diff
@@ -144,18 +140,33 @@ export const getContractTypesConfig: TGetContractTypesConfig = symbol => ({
         ],
         config: { hide_duration: true },
     }, // hide Duration for Multiplier contracts for now
+    turboslong: {
+        title: localize('Long/Short'),
+        trade_types: ['TURBOSLONG'],
+        basis: ['stake'],
+        barrier_count: 1,
+        components: ['trade_type_tabs', 'barrier_selector', 'take_profit'],
+    },
+    turbosshort: {
+        title: localize('Long/Short'),
+        trade_types: ['TURBOSSHORT'],
+        basis: ['stake'],
+        barrier_count: 1,
+        components: ['trade_type_tabs', 'barrier_selector', 'take_profit'],
+    },
     vanilla: {
         title: localize('Call/Put'),
         trade_types: ['VANILLALONGCALL', 'VANILLALONGPUT'],
         basis: ['stake'],
-        components: ['duration', 'strike', 'amount', 'vanilla_trade_type'],
+        components: ['duration', 'strike', 'amount', 'trade_type_tabs'],
         barrier_count: 1,
         config: { should_override: true },
     },
 });
 
 // Config for rendering trade options
 export const getContractCategoriesConfig = () => ({
+    Turbos: { name: localize('Turbos'), categories: ['turboslong', 'turbosshort'] },
```

Comment:
I see, `turboslong` and `turbosshort` hardcoded in multiple places, perhaps as part of the improvements in future, we can define a constant and reuse it.

Reply:
@matin-deriv sure, we can do it later for all types of contracts listed here👍🏻

Reply:
@kate-deriv has done it👍🏻